dickdavis/event_logger_rails
# Fix event logger output and engine initialization

## Description

This PR fixes the event logger output and engine initialization. As a result of the new setup, an initializer is no longer necessary. Also, the registered events can now vary between Rails environments.

## Testing

Follow these steps to test this PR:

* Add the gem to the Gemfile of a test application, pointing to this branch.
* Bundle install.
* Add a `log_event` somewhere with an app-specific event.
* Verify it logs to the log file in the correct format.
